libminifi/src/minifi-c.cpp

```diff
@@ -369,8 +377,8 @@ MinifiLogLevel MinifiLoggerLevel(MinifiLogger logger) {
   gsl_FailFast();
 }
 
-OWNED MinifiPublishedMetrics MinifiPublishedMetricsCreate(uint32_t count, const MinifiStringView* names, const double* values) {
-  auto* metrics = new std::vector<minifi::state::PublishedMetric>();
+OWNED gsl::owner<MinifiPublishedMetrics> MinifiPublishedMetricsCreate(const uint32_t count, const MinifiStringView* names, const double* values) {
+  const gsl::owner<std::vector<minifi::state::PublishedMetric>*> metrics = new std::vector<minifi::state::PublishedMetric>();  // NOLINT(modernize-use-auto)
```

Comment:
this one... if you use auto it complains it you are using raw pointers without gsl owner, if you are explicit like this then it complains why not use auto